src/main/java/net/openhft/chronicle/wire/GenerateMethodReader.java

```diff
@@ -316,10 +319,7 @@ private void generateSourceCode() {
                 "throw e;\n" +
                 "}\n" +
                 "catch (Exception e) {\n" +
-                "Jvm.warn().on(this.getClass(), \"Failure to dispatch message, " +
-                "will retry to process without generated code: \" + lastEventName + \"(), " +
-                "bytes: \" + wireIn.bytes().toDebugString(), e);\n" +
-                "return false;\n" +
+                "throw new InvocationTargetRuntimeException(e);\n" +
```

Comment:
I'm not 100% sure about throwing this - maybe we should just not catch this exception?